webapp/src/components/backstage/playbook_runs/playbook_run/rhs_info_activity.tsx

```diff
@@ -3,17 +3,78 @@
 
 import React from 'react';
 import {useIntl} from 'react-intl';
+import styled from 'styled-components';
+import {DateTime} from 'luxon';
 
+import {useSelector} from 'react-redux';
+import {GlobalState} from '@mattermost/types/store';
+import {getChannelsNameMapInCurrentTeam} from 'mattermost-redux/selectors/entities/channels';
+import {Team} from '@mattermost/types/teams';
+import {getTeam} from 'mattermost-redux/selectors/entities/teams';
+
+import {ChannelNamesMap} from 'src/types/backstage';
 import {Section, SectionTitle} from 'src/components/backstage/playbook_runs/playbook_run/rhs_info_styles';
+import {PlaybookRun} from 'src/types/playbook_run';
+import TimelineEventItem from 'src/components/backstage/playbook_runs/playbook_run_backstage/retrospective/timeline_event_item';
+import {clientRemoveTimelineEvent} from 'src/client';
+
+import {useTimelineEvents} from './timeline_utils';
+
+const SHOWED_EVENTS = 5;
 
-const RHSInfoActivity = () => {
+interface Props {
+    run: PlaybookRun;
+}
+
+const RHSInfoActivity = ({run}: Props) => {
     const {formatMessage} = useIntl();
+    const [filteredEvents] = useTimelineEvents(run);
+    const channelNamesMap = useSelector<GlobalState, ChannelNamesMap>(getChannelsNameMapInCurrentTeam);
+    const team = useSelector<GlobalState, Team>((state) => getTeam(state, run.team_id));
```

Comment:
Nit: Do we need these types?
```suggestion
    const channelNamesMap = useSelector(getChannelsNameMapInCurrentTeam);
    const team = useSelector((state: GlobalState) => getTeam(state, run.team_id));
```